tests/phpunit/tests/theme.php

```diff
@@ -289,12 +289,38 @@ public function test_switch_theme() {
 				$this->assertNotEmpty( $root_uri );
 
 				$this->assertSame( $root_fs . '/' . get_stylesheet(), get_stylesheet_directory() );
```

Comment:
The assertions earlier in this test method should also have `message` parameters.

Reply:
Yes they should. But I think that needs to be in a follow-up PR for test improvements. I chose not to update the untouched assertions in this test to minimize the number of changes in this PR.